### ISSUE TITLE
tests/test_benchmark.py run_binaryen_opts() composes an impossible path

### ISSUE THREAD

**Issue by jonassmedegaard** [emscripten-core/emscripten#14724]
file `tests/test_benchmark.py` contains this:
```python
os.path.join(building.get_binaryen_bin(), 'wasm-opt', '--all-features')
```

I am not strong with python and did not actually execute the code, but it seems to produce an impossible path `/usr/bin/wasm-opt/--all-features`.

The option should be outside the parense, like this:
```python
os.path.join(building.get_binaryen_bin(), 'wasm-opt'), '--all-features'
```

Perhaps noone has used that script with binaryen_opts set?

**Comment by sbc100** [comment]
Yes probably just a bug.   That particular script tends to bitrot since its not actually run continuously anywhere, and only run locally by a few folks when they need it.
